css/index.css

```diff
@@ -1,3 +1,338 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+
+.container{
+	
+    width: 800px;
+    margin: auto;
+	border: 1px solid white;
+}
+
+@media(max-width: 500px) {
+      .container{
+          display: flex;
+          flex-direction: column; 
+          align-content: center; 
+          justify-content: center;
+          max-width: 500px;
+          width:auto;  
+          margin:auto; 
+      }
+
+      .cta img{
+          display:none;
+      }
+      header{
+          display:flex;
+          align-items: center;
+          width:auto;
+          padding-bottom: 8%
+          
+      }
+      nav{
+          display: flex;
+          flex-direction: column-reverse;
+          align-content: center;
+          margin-left: 25%;
+          margin-right: 20%;
+      }
```

Comment:
<h3> I would recommend playing with your nav styles for mobile to try to get the nav to stretch the way it does in the design file. But your are definitely doing an awesome job at using queries!